build-tools-internal/src/main/resources/changelog-schema.json

```diff
@@ -21,6 +21,7 @@
             "Aggregations",
             "Allocation",
             "Analysis",
+            "Application",
```

Comment:
nit: can we call it "Enterprise Search" to match the github label?

Reply:
Shouldn't we abandon Enterprise Search as a name eventually, and focus on Seach Applications? 🤔

Reply:
➕ ➕ @carlosdelest